### PR TITLE
Introduce new `directus` wrapper package

### PR DESCRIPTION
**Pull request by paescuj**
Introducing a new `directus` wrapper package while renaming the old `directus` package to `@directus/api`.
This gives us more flexibility in versioning the sub-packages individually, keeping the main version in `directus` and enables us to adhere to separation of concerns (also making it more obvious that `@directus/api` contains - you name it - the api).

### CODE REVIEW

File: api/package.json [hunk 1]
```diff
@@ -68,7 +66,6 @@
 		"build": "tsc --build && copyfiles \"src/**/*.{yaml,liquid}\" -u 1 dist",
 		"cli": "NODE_ENV=development SERVE_APP=false tsx src/cli/run.ts",
 		"dev": "NODE_ENV=development SERVE_APP=false tsx watch --clear-screen=false src/start.ts",
-		"start": "node cli.js start",
```
Comment (paescuj) [inline api/package.json:71]: Should we add this to the new wrapper package?

File: api/package.json [hunk 2]
```diff
@@ -229,6 +226,6 @@
 		"tedious": "15.1.3"
 	},
 	"engines": {
-		"node": ">=12.20.0"
+		"node": ">=18.0.0"
```
[the remark this answers is not in the document]
Comment (rijkvanzanten) [reply api/package.json:229]: It should be 18+ everywhere

File: directus/package.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+		}
+	],
+	"exports": {
+		"./package.json": "./package.json"
+	},
+	"bin": {
+		"directus": "cli.js"
+	},
+	"dependencies": {
+		"@directus/api": "workspace:9.24.0"
```
Comment (rijkvanzanten) [inline directus/package.json:60]:
```suggestion
		"@directus/api": "workspace:*"
```

File: directus/package.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+		"vue"
+	],
+	"homepage": "https://directus.io",
+	"bugs": {
+		"url": "https://github.com/directus/directus/issues"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/directus/directus.git",
+		"directory": "directus"
```
Comment (rijkvanzanten) [inline directus/package.json:32]:
```suggestion
```

**Review by rijkvanzanten — APPROVED** [review]

File: api/package.json (hunk 1 above)
Comment (paescuj) [reply api/package.json:71]: Hmm, yeah I don't think so too. Let's leave it out completely 👍

**Pull request merged**